framework/wazuh/core/manager.py

```diff
@@ -35,6 +35,7 @@
 WAZUH_UID_KEY = 'wazuh-uid'
 WAZUH_TAG_KEY = 'wazuh-tag'
 USER_AGENT_KEY = 'user-agent'
+DEFAULT_TIMEOUT = 10.0
```

Comment:
Add a new line to conform with the PEP8 style guide (classes should be separed by two empty lines)

```suggestion
DEFAULT_TIMEOUT = 10.0

```